test/tests/server_connectorTest.js

```diff
@@ -285,4 +285,29 @@ describe("Connector Server", function () {
 			);
 		});
 	});
+	
+	describe('/connector/getTranslatorCode', function() {
+		it('should respond with translator code', function* () {
+			var code = 'function detectWeb() {}\nfunction doImport() {}';
+			var translator = buildDummyTranslator(4, code);
+			sinon.stub(Zotero.Translators, 'get').returns(translator);
+			
+			var response = yield Zotero.HTTP.request(
+				'POST',
+				connectorServerPath + "/connector/getTranslatorCode",
+				{
+					headers: {
+						"Content-Type": "application/json"
+					},
+					body: JSON.stringify({
+						translatorID: "dummy-translator",
+					})
+				}
+			);
+			
+			assert.equal(response.response, code);
+			
+			Zotero.Translators.get.restore();
+		})
+	});
```

Comment:
Might as well move this above the other two `describe` blocks to be both alphabetically and chronologically correct.